controllers/userController.js

```diff
@@ -0,0 +1,108 @@
+const userService = require("../services/dbUserServices");
+
+class UserController {
+  async getAllUsers(req, res) {
+    try {
```

Comment:
No need to use `try-catch` because you've already got error handler

Reply:
error handler catch all unhandled error and answer for mistake is very common. Is it a good practice?